compiler/control/OMROptions.cpp

```diff
@@ -3602,6 +3602,7 @@ OMR::Options::jitPostProcess()
       {
       TR::Compiler->host.setSMP(false);
       TR::Compiler->target.setSMP(false);
+      TR::Compiler->relocatableTarget.setSMP(false);
```

Comment:
I have a strong opinion about this: SMP should be set to true to avoid cases when code is generated assuming uniprocessors and then used in a multiprocessor environment.

Reply:
Are you suggesting we should have SMP set to `true` at all times for `TR::Compiler->relocatableTarget`?

Reply:
SMP is set to true by default and only set to false with `TR_ForceNonSMP`. Are you saying `TR_ForceNonSMP` should not apply to relocatableTarget environment?

Reply:
Ok. My browser did not show the `if` condition. If SMP ends up being `true` by default I am happy.